src/main/java/Task.java

```diff
@@ -0,0 +1,30 @@
+public class Task {
+
+    String name;
```

Comment:
Since you have a `getDescription()` method, would it be a good idea to set name as private? Also maybe we can also name it something else rather than `name` so it is more descriptive?

Reply:
good idea, setting to private and changing to description instead of name